Eugene-CG/popup/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: content-box;
+	list-style: none;
+    outline: none;
+}
+.wrapper {
+	width: 100%;
+	height: 300vh;
+	background-image: url(./img/gitlogoforme.jpeg);
+}
+.header {
+    margin: 0 auto;
+	max-width: 1198px;
+	height: 80px;
+	display: flex;
+	justify-content: space-between;
+}
+img {
+	width: 50px;
+	height: 50px;
+}
+a {
+	display: block;
+}
+.list {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	height: 80px;
+}
+li {
+	margin: 0.2em;
+}
+.list__item {
+	margin: 10px 20px;
+}
+.popup, .popup-more {
+	align-self: flex-start;
+	position: relative;
+}
+a, span {
+	text-decoration: none;
+	color: rgb(255, 255, 255);
+	line-height: 1.4;
+	font-weight: 700;
+	font-size: 1.3em;
+}
+a {
+    width: 50px !important;
+}
+.popup-btn__label, .more-btn__label {
+	margin: 6em;
+}
+.popup-btn, .more-btn {
+	outline: none;
+	position: absolute;
+	appearance: none;
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	top: 0;
+	left: 0;
+}
+.popup-btn {
+	width: 50px;
+	height: 50px;
+}
+.more-btn {
+	width: 48.13px;
+	height: 24px;
+}
+.popup-list, .more-list {
+	overflow: hidden;
+	height: 0px;
+    width: 67px;
+    padding-right: 17px;
+}
+.more-list {
+	visibility: hidden;
+}
+.popup-btn:checked +.popup-list{
+	overflow-y: auto;
+    height: 336.5px;
```

Comment:
Please, fix tab spaceing and rules ordering.

Reply:
done